frontend/src/use-cases/songs/Songs.context.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { createContext, useContext, useReducer } from "react";
+import { getSongs } from "../../api/songs/get.songs.api";
+import { getTags } from "../../api/tags/get.tags.api";
+
+export const SongTagContext = createContext({});
+
+export const SongTagProvider = ({ reducer, initialState, children }) => (
```

Comment:
I like to throw in the `Singleton` in the name to signal that this should only be used once in the app.

Reply:
i think it be fine